modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -403,6 +446,80 @@ Users who can access the Windows Network Drive storage via share won't receive a
 
 NOTE: `wnd.activity.sendToSharees` key depends on the `wnd.activity.registerExtension` key to take effect.
 
+=== Collaborative WND
+
+CWND can only be set by an admin in menu:Settings[Admin > Storage]. This mount type cannot be selected by users in the user section. To prepare access for your mount point using the CWND mount type, you must provide a _Service Account_ (SA) which is an oridnary SMB user granting read access to the share you want to mount. You can use one SA for all CWND mounts or separate ones. The SA is used to gather the contents of a share, used by the WND Listener and provides a common `file_id` to all accessing users, while the accessing user has only access to those files and folders he is granted for.
+
+. As an admin, goto menu:Settings[Admin > Storage] and create a new CWND based mount point.
++
+.Figure 5. Add a Collaborative Windows Network Drive Mount
+image:enterprise/external_storage/windows_network_drive/cwnd_add_storage.png[Add Collaborative Windows Network Drive Mount, width=50%]
+. Chose any name for the mount point that fits your needs.
+. Select User Login Type.
++
+[IMPORTANT]
+====
+Only the following three are valid selections for CWND
+[loweralpha]
+. `Log-in credentials, saved in session`
+. `Log-in credentials, saved in database`
+. `User entered, stored in database` ^[1]^
++
+[1] Must be used in case user authentication is made with OIDC
+====
++
+.Figure 6. Select How User Logs Into the Mount Point
+image:enterprise/external_storage/windows_network_drive/cwnd_login_possibilities.png[Select How User Logs Into the Mount Point]
+[loweralpha]
+.. `Log-in credentials, saved in session`
++
+When the user logs into ownCloud via browser, the credentials to authenticate CWND are taken from this login. These credentials immediate end when the user logges out because the session has ended. This may impact client synchronisation or any access to other users via sharing. In case, no synchroinisation can be processed and sharees will lose access tho their shared resource when the sharer logges out. _This login type is by design not compatible with OIDC authentication._
```

Comment:
"no synchronisation can be processed and sharees will lose access to"

Reply:
> In case, no synchroinisation can be processed and sharees will lose access tho their shared resource when the sharer logges out

It seems to suggest that the access is possible while the user is logged in, but this isn't true. Even if userA is logged in, neither userB nor ownCloud will be able to access to userA's credentials, so nobody can access to the storage on behalf of userA (shares, file scan, etc)

Reply:
If I am userA and share my mount (or a subfolder of my mount) with userB,
userB accesses the mount with userAs credentials - this is my understanding - am I wrong?

Reply:
That's right. However, userB won't be able to access to userA's credentials in this case even if userA is logged in because userB cannot access to the session information of userA

Reply:
Waiting for input (see chat)

Reply:
got the input, working on it

Reply:
fixed